src/pkg/packaging/rpm/scripts/after_install_host.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+echo "Creating dotnet host symbolic link: /usr/bin/dotnet"
+ln -sf "/usr/share/dotnet/dotnet" "/usr/bin/dotnet"
```

Comment:
I think newer versions of fpm support symlinks (looking over their issues it was possible there was a bug at some point).  We should try to move to that at some point (which would bring things inline with what we do in debian land).

Probably best to file and issue and then we can do an updated PR next.